VultisigApp/VultisigApp/Chains/dydx.swift

```diff
@@ -20,7 +21,7 @@ class DydxHelper {
     static let DydxGasLimit:UInt64 = 2500000000000000
     
     func getSwapPreSignedInputData(keysignPayload: KeysignPayload,signingInput: CosmosSigningInput) -> Result<Data,Error> {
-        guard case .Cosmos(let accountNumber, let sequence,let gas) = keysignPayload.chainSpecific else {
+        guard case .Cosmos(let accountNumber, let sequence,let gas, let isDeposit) = keysignPayload.chainSpecific else {
```

Comment:
shouldn't here also be `transactionType`?

Reply:
I’ll double check, tks